localtypings/pxtarget.d.ts

```diff
@@ -769,6 +769,7 @@ declare namespace pxt.tutorial {
         tutorialStepInfo?: pxt.tutorial.TutorialStepInfo[];
         tutorialStep?: number; // current tutorial page
         tutorialReady?: boolean; // current tutorial page
+        tutorialHintCounter?: number // count for number of times hint has been shown
```

Comment:
this flag gets serialized in the header... you should probably clear it when loading the script. or store it somewhere else. I think the limit should be per "editor session".